samples/Notifications.sarif

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "properties": {
+    "comment": "This sample illustrates tool configuration and execution notifications."
+  },
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SarifSamples",
+          "rules": [
+            {
+              "id": "TUT1201",
+              "defaultConfiguration": {
+                "level": "error"
+              },
+              "messageStrings": {
+                "default": {
+                  "text": "The statement does not end with a semicolon."
+                }
+              }
+            }
+          ],
+          "notifications": [
+            {
+              "id": "TUTN9001",
+              "name": "unknown-rule",
+              "defaultConfiguration": {
+                "level": "warning"
+              },
+              "shortDescription": {
+                "text": "This notification is triggered when the user supplies a command line argument to enable or disable a rule that does not exist."
+              },
+              "messageStrings": {
+                "disabled": {
+                  "text": "The rule '{0}' cannot be disabled because it does not exist."
+                },
+                "enabled": {
+                  "text": "The rule '{0}' cannot be enabled because it does not exist."
+                }
+              }
+            },
+            {
+              "id": "TUTN9002",
+              "name": "rule-threw-exception",
+              "defaultConfiguration": {
+                "level": "error"
+              },
+              "shortDescription": {
+                "text": "This notification is triggered when an analysis rule throws an exception."
+              },
+              "fullDescription": {
+                "text": "This notification is triggered when an analysis rule throws an exception while analyzing a file. Depending on the command line options, the rule might either be disabled, or it might continue to run on subsequent files."
+              },
+              "messageStrings": {
+                "disable": {
+                  "text": "The rule '{0}' threw a '{1}' while analyzing file '{2}'. The rule has been disabled."
```

Comment:
>'{0}' threw a '{1}' while analyzing file '{2}'. The rule has been disabled [](start = 36, length = 74)

'{0}' threw a '{1}' exception while analyzing '{2}'. The rule has been disabled.

suggest same pattern for continue